src/components/App/App.jsx

```diff
@@ -26,16 +25,23 @@ class App extends React.Component {
     ipcRenderer.send('title', 'Notifications')
     this.setupAppMenu()
     if (GitHubAuth.isAuthenticated()) {
+      this.loadUser()
       const key = LastFilter.retrieve()
       if (key) {
         this.loadFilter(key)
       } else {
         this.manageFilters()
       }
-      this.loadUser()
     }
   }
 
+  onUserLoad(user) {
+    this.setState({ user })
+    const filters = new DefaultFilters(user.login)
+    filters.addDefaults()
+    this.setState({ filters: Filters.findAll() })
```

Comment:
Could wait and combine both the new state properties. Remove line 39 and do `this.setState({ filters: Filters.findAll(), user })` here.